src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
```

Comment:
The `ReturnOperation` class should be placed in the `strategy` package to follow the project structure requirements from the checklist.